server/game/GameActions/TransferHonorAction.ts

```diff
@@ -29,18 +29,39 @@ export class TransferHonorAction extends PlayerAction {
 
     canAffect(player: Player, context: AbilityContext, additionalProperties = {}): boolean {
         let properties = this.getProperties(context, additionalProperties) as TransferHonorProperties;
-        return player.opponent && properties.amount > 0 && super.canAffect(player, context);
+
+        const gainsHonor = properties.amount > 0;
+        if(!gainsHonor)
+            return false;
+
+        const honorGainLimitPerPhase = player.opponent.getEffects && player.opponent.getEffects(EffectNames.LimitHonorGainPerPhase).reduce((total, value) => total + value, 0);
+        const honorGainedThisPhase = player.opponent.honorGained && player.opponent.honorGained(context.game.roundNumber, context.game.currentPhase, true);
+        if(honorGainLimitPerPhase && honorGainedThisPhase >= honorGainLimitPerPhase)
+            return false;
```

Comment:
If we're duplicating this between actions should we move it to a method on player?